README.md

```diff
@@ -186,14 +186,40 @@ export interface Styles {
 }
 
 export type ClassNames = keyof Styles;
-
```

Comment:
This felt too "spacey" for me, but let me know if you want me to add back the spaces here and in the generator.

Reply:
Thanks for the feedback. @skovy I'd really like your opinion on whether or not I should put back the newline spacing here.

Reply:
Since it's not a functional change it doesn't really matter and I don't feel too strongly so I think this is fine. 

Since you asked, in general, I'm a fan of single-change PRs to make reviews easier. If this did for whatever reason cause an issue it's easy to revert only this change. 

Also, I'm a fan of whitespace and think of whitespace in a similar way as to paragraphs. I like to group similar things and separate unrelated things by whitespace. You could say these are all for classes (all the same, no spaces), or you could say this is a list of class names, while the next thing is an interface of all the classnames, while the last is an actual object of classes. I personally would go with the latter and consider these slightly different concepts and separate them with whitespace.

But again, for the purposes of this, I think this change is fine.

Reply:
Thanks for the input. That makes sense. I reverted the newline change.